packages/bee-q/src/components/option/_storybook/bq-option.stories.tsx

```diff
@@ -0,0 +1,94 @@
+import type { Args, Meta, StoryObj } from '@storybook/web-components';
+
+import { html } from 'lit-html';
+
+import mdx from './bq-option.mdx';
+
+const meta: Meta = {
+  title: 'Components/Option',
+  component: 'bq-option',
+  parameters: {
+    docs: {
+      page: mdx,
+    },
+  },
+  argTypes: {
+    disabled: { control: 'boolean' },
+    selected: { control: 'boolean' },
+    // Event handlers
+    bqOptionBlur: { action: 'bqOptionBlur' },
+    bqOptionFocus: { action: 'bqOptionFocus' },
+    bqOptionClick: { action: 'bqOptionClick' },
+    // Not part of the public API, so we don't want to expose it in the docs
+    text: { control: 'text', table: { disable: true } },
+    htmlNodePrefix: { control: 'object', table: { disable: true } },
+    htmlNodeSuffix: { control: 'object', table: { disable: true } },
+  },
+  args: {
+    disabled: false,
+    selected: false,
+    text: 'Option label',
+    htmlNodePrefix: Object.assign(document.createElement('bq-icon'), {
+      name: 'user',
+      size: '16',
+      slot: 'prefix',
+    }),
+    htmlNodeSuffix: Object.assign(document.createElement('bq-icon'), {
+      name: 'gear',
+      size: '16',
+      slot: 'suffix',
+    }),
+  },
+};
+export default meta;
+
+type Story = StoryObj;
+
+const Template = (args: Args) => html`
+  <bq-option-list>
+    <bq-option
+      disabled=${args.disabled}
+      selected=${args.selected}
```

Comment:
Boolean attributes should be bound with a `?` in front: https://lit.dev/docs/v1/lit-html/template-reference/#binding-types

```suggestion
      ?disabled=${args.disabled}
      ?selected=${args.selected}
```